src/client/create-client.ts

```
@@ -15,8 +14,13 @@ import type {
 import { ConvexError, v } from "convex/values";
 import type { Infer } from "convex/values";
 import { convexAdapter } from "./adapter.js";
+import {
+  createTypedAdapter,
+  type AdapterFunctions,
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)